src/web_interface/templates/home.html

```diff
@@ -2,6 +2,30 @@
 
 {% set active_page = "Home" %}
 
+{% block head %}
```

Comment:
We should discuss at which point we move all the custom css stuff we do into it's own `fact.css`

Reply:
agreed